### DISCUSSION TITLE
configure with --host=win32 problem with slash

### DISCUSSION THREAD

**Discussion by agracia-saes** [OpenDDS/OpenDDS#4613] [Q&A]
Hello, when I exec configure with --host=win32 option I have a error with slash.

ERROR: Can't write to \c\LIB\OpenDDS-3.28\ACE_wrappers/ace/config.h, stopped at configure line 1091.

And it's not continue

**Comment by mitza-oci** [comment]
`\c\LIB\` doesn't look like a Windows path.  You don't need `--host=win32` when configuring on Windows since it's a Windows Perl interpreter and the default works.

Please see https://opendds.readthedocs.io/en/latest-release/devguide/quickstart/windows.html
